services/processor/native/repository/Committee/update.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 the orbs-network-go authors
+// This file is part of the orbs-network-go library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package committee_systemcontract
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/env"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/events"
+)
+
+func updateReputation() {
```

Comment:
As for your third comment from the email - we should check signer/trigger for updateReputation so to permit only "system access" => empty signer ..